projects/dot-rules/src/lib/rule-action-component.ts

```diff
@@ -55,18 +55,13 @@ export class RuleActionComponent {
 
     @Input() typeDropdown: any;
 
-    constructor(private _resources: I18nService, private loggerService: LoggerService) {}
+    constructor(private _resources: I18nService, private loggerService: LoggerService) {
+        this.typeDropdown = {
```

Comment:
Why don't just init in the property itself? is not recommend it do this kind of stuff in the constructor: https://angular-checklist.io/default/checklist/components/ZXTLWo